.gitignore

```diff
@@ -8,6 +8,7 @@
 # testing
 /coverage
 .nyc_output
+/cypress
```

Comment:
Why is a cypress folder being generated?

Reply:
i test the downloads in devfile-page.spec.ts, and in order to do so, cypress creates a cypress/downloads folder

Reply:
Hmm ok. I wonder if there is a setting that can disable the download from being stored

Reply:
I guess there is no good way to turn off the download from being stored after it runs. It may be better to store it in a temp folder? Your choice.

Reply:
https://docs.cypress.io/guides/references/configuration#Downloads